components/select/nz-select.component.html

```diff
@@ -16,7 +16,22 @@
     [nzListTemplateOfOption]="listOfTemplateOption"
     [nzListOfSelectedValue]="listOfSelectedValue"
     (nzOnSearch)="onSearch($event.value,$event.emit)"
-    (nzListOfSelectedValueChange)="updateListOfSelectedValueFromTopControl($event)">
+    (nzListOfSelectedValueChange)="updateListOfSelectedValueFromTopControl($event)"
+    *ngIf="nzAllowTitle" title="{{value}}">
+  </div>
+  <div
+    nz-select-top-control
+    [nzOpen]="nzOpen"
+    [compareWith]="compareWith"
+    [nzPlaceHolder]="nzPlaceHolder"
+    [nzShowSearch]="nzShowSearch"
+    [nzDisabled]="nzDisabled"
+    [nzMode]="nzMode"
+    [nzListTemplateOfOption]="listOfTemplateOption"
+    [nzListOfSelectedValue]="listOfSelectedValue"
+    (nzOnSearch)="onSearch($event.value,$event.emit)"
+    (nzListOfSelectedValueChange)="updateListOfSelectedValueFromTopControl($event)"
+    *ngIf="!nzAllowTitle">
```

Comment:
This behavior should be default, no need `nzAllowTitle` input.

Reply:
It seems to be a little bit problem.for simplly,I just wanna set a defalut property 'title' to the 'div' by setting nzAllowTitle's value.Do u have any ideas to provide for me?

Reply:
@Picktarget Should always has the title property, no need `nzAllowTitle`.